sdk/aqueduct/flow_run.py

```diff
@@ -64,6 +68,26 @@ def describe(self) -> None:
             assert param_op.spec.param is not None, "Artifact is not a parameter."
             print("* " + param_op.name + ": " + param_op.spec.param.val)
 
+    def artifact(
+        self, name: str
+    ) -> Optional[Union[TableArtifact, MetricArtifact, CheckArtifact, ParamArtifact]]:
+        """Gets the Artifact from the flow based on artifact_name"""
+        flow_run_dag = self._dag
+        artifact_from_dag = flow_run_dag.get_artifacts_by_name(name)
+
+        if artifact_from_dag is None:
+            raise ArtifactNotFoundException("The artifact name provided does not exist.")
```

Comment:
I wonder if it might be nicer for the user if we return None if the artifact doesn't exist? Maybe they just want to check for existence of an artifact, and its a bit less annoying to check for None than it is to catch an error.

Reply:
If you add that, could you also put that in the function comment string?

Reply:
I think that's a good idea!